nncf/openvino/quantization/quantize.py

```diff
@@ -130,13 +127,46 @@ def quantize_impl(model: ov.Model,
         }
     ]
 
+    algo_names = sorted([x['name'] for x in algorithms])
```

Comment:
Now half of the code of this method is about sending telemetry. I wonder if there is a more elegant way to do that, maybe with `with` operator.

Reply:
It can save space on session starting and ending code, but not on the intermediate event sending.